src/storage/store/utils.rs

```diff
@@ -1,3 +1,4 @@
+pub const PAGE_SIZE_MAX: usize = 10_000;
```

Comment:
This is fine for messages, but way too high for blocks and shards, should be more like 100-1000 for them